ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/InsertValuesExecutorTest.java

```diff
@@ -1121,7 +1121,9 @@ private void givenDataSourceWithSchema(
           schema,
           Optional.empty(),
           false,
-          topic
+          topic,
+          false,
+          false
```

Comment:
Does this PR add a test for rejected `INSERT INTO` for source tables? Should it? Or do we have some other work item for it?

Reply:
This PR does not add it. Will be in a follow-up.